tests/filestore-filecontainer-http/test.yaml

```diff
@@ -2,8 +2,7 @@ requires:
   features:
     - HAVE_NSS
     - MAGIC
-  files:
-    - src/output-filestore.c
```

Comment:
@jasonish have you removed all the files options because the features are guaranteed to be in the specified minimum versions? If yes, should `files` option be removed altogether since all such cases can be covered by `min-version` or `version`. If no, please let me know what's the thought behind this.

Reply:
I have a use for this for long running dev branches that remain out of tree for a long time.

Reply:
Or even features that are added, but we want to have tests for without having had a version bump in the master branch yet.

Reply:
> have you removed all the files options because the features are guaranteed to be in the specified minimum versions?

Yes. Using 4.1 as the lowest supported version now.